examples/jsm/loaders/GLTFLoader.js

```diff
@@ -4479,18 +4480,27 @@ class GLTFParser {
 
 		const extensions = this.extensions;
 		const sceneDef = this.json.scenes[ sceneIndex ];
+		const nodeIds = sceneDef.nodes || [];
 		const parser = this;
+		const isSingleRoot = this.json.extensionsUsed.includes( EXTENSIONS.GODOT_SINGLE_ROOT );
```

Comment:
The loader is currently broken for certain glTF assets because of this line. Assets which use no extensions like in the `webgl_loader_gltf` demo have no `extensionsUsed` property defined in their JSON. Below code should do the trick:
```js
const extensionsUsed = json.extensionsUsed;
const isSingleRoot = Array.isArray( extensionsUsed ) ? extensionsUsed.includes( EXTENSIONS.GODOT_SINGLE_ROOT ) : false;
```

Reply:
Ah, that makes a lot of sense. I'll use the code in your suggestion, but it also occurred to me to use `const extensionsUsed = this.json.extensionsUsed || [];`.